ecosystem/sep-0012.md

```diff
@@ -188,14 +217,16 @@ REJECTED | This customer's KYC has failed and will never succeed.  The `message`
 
 #### Fields
 
-The fields entry is required for the `NEEDS_INFO` status.  Fields should be specified as an object with keys representing field names required, preferably from SEP-9.
+The fields entry is required for the `NEEDS_INFO` status but may be included with any status. Fields should be specified as an object with keys representing the [SEP-9](sep-0009.md) field names required.
 
 Property | Type | Description
 -----|------|------------
 `type` | enum | The data type of the field value. Can be `string`, `binary`, `number`, or `date`
 `description` | string | A human-readable description of this field, especially important if this is not a SEP-9 field.
 `choices` | array | (optional) An array of valid values for this field.
 `optional` | boolean | (optional) A boolean whether this field is required to proceed or not. Defaults to false.
+`status` | string | (optional) One of `NOT_PROVIDED`, `PROCESSING`, `UNVERIFIED`, `ACCEPTED`, `REJECTED`. See [PUT /customer/verification](#customer-put-verification) for more information on `UNVERIFIED` fields.
```

Comment:
❗   What is the `UNVERIFIED` state? It doesn't look like all the statuses are defined:

https://github.com/stellar/stellar-protocol/blob/6320ef8124c85805206045ec31448358aa4d656b/ecosystem/sep-0012.md#L209-L216

💡 It's quite difficult to ascertain the meaning of UNVERIFIED in the context of the other statuses. It sounds like it could be the same thing as PROCESSING. I think we should find another name for it, maybe `VERIFICATION_REQUIRED` which signals a little more explicitly that there is some action required by the user. We could also name it `ACTION_REQUIRED` to generalize it a little more and be more explicit, but `VERIFICATION_REQUIRED` I think is better because it uses the same terms as the new verification endpoint.

Reply:
I added a `Field Statuses` section, and renamed `UNVERIFIED` to `VERIFICATION_REQUIRED`.